shipment.go

```diff
@@ -345,9 +349,24 @@ func (c *Client) RerateShipmentWithCarbonOffsetWithContext(ctx context.Context,
 	return
 }
 
+// BetaGetStatelessRates fetches a list of stateless rates for a proposed shipment, without creating a shipment object.
+func (c *Client) BetaGetStatelessRates(in *Shipment) (out []*StatelessRate, err error) {
+	return c.BetaGetStatelessRatesWithContext(context.Background(), in)
+}
+
+// BetaGetStatelessRatesWithContext performs the same operation as BetaGetStatelessRates,
+// but allows specifying a context that can interrupt the request.
+func (c *Client) BetaGetStatelessRatesWithContext(ctx context.Context, in *Shipment) (out []*StatelessRate, err error) {
+	req := &createShipmentRequest{Shipment: in}
+	res := &getShipmentStatelessRatesResponse{Rates: &out}
+	err = c.post(ctx, "/beta/rates", &req, &res)
+	return
+}
+
 // Deprecated: Use LowestShipmentRate instead.
 // LowestRate gets the lowest rate of a shipment
 func (c *Client) LowestRate(shipment *Shipment) (out Rate, err error) {
+	// TODO: Why did we deprecate this and replace it with an arguably worse name?
```

Comment:
Please remove these TODOs. Rants shouldn't belong in code.

We deprecated the generically named lowest rate functions because we now support lowest rate for pickups and orders too. Because functions in Go aren't tied to an object or class, we needed to distinguish them via the function name.